Language/GLSL/Pretty.hs

```diff
@@ -73,6 +73,13 @@ instance Pretty Declaration where
   pPrint (Precision pq t) = text "precision" <+> pPrint pq <+> pPrint t <> semi
   pPrint (Block tq i ds n) = vcat [pPrint tq <+> text i, lbrace, nest 2 (vcat $ map pPrint ds), rbrace <+> ident n <> semi]
   pPrint (TQ tq) = pPrint tq <> semi
+  pPrint (ExtensionDirective name behavior) = text "#extension " <+> text name <+> text " : " <+> pPrint behavior
```

Comment:
I think `<+>` takes care of adding the white space, and that before the colon, we should use only `<>`.

Reply:
Thanks, should be fixed now